node.gyp

```diff
@@ -747,6 +749,48 @@
   ], # end targets
 
   'conditions': [
+    [ 'node_target_type=="static_library"', {
+      'targets': [
+        {
+          'target_name': 'static_node',
+          'type': 'executable',
+          'product_name': '<(node_core_target_name)',
+          'dependencies': [
+            '<(node_core_target_name)',
+          ],
+          'sources+': [
+            'src/node_main.cc',
+          ],
+          'include_dirs': [
+            'deps/v8/include',
+          ],
+          'xcode_settings': {
+            'OTHER_LDFLAGS': [
+              '-Wl,-force_load,<(PRODUCT_DIR)/<(STATIC_LIB_PREFIX)'
+              '<(node_core_target_name)<(STATIC_LIB_SUFFIX)',
```

Comment:
Maybe indent this by 4 spaces to make it clear it's intentionally a line continuation (likewise below.)

Reply:
Yeah those lines should be indented. I'll fix them.